Build script doesn't support python2

If python3 hasn't been installed on MacOS (e.g. via homebrew), the build script fails with the following output:

```-- [pmm] No existing Conan installation found. We'll try to obtain one.
-- [pmm] Get Conan with Python 2
-- [pmm] Get Conan with Python 2 - Candidate Python: /usr/bin/python2.7 
-- [pmm] Get Conan with Python 2 - Fail: No virtualenv module
CMake Error at build/_pmm/1.3.0/conan.cmake:586 (message):
  Cannot use Conan with PMM because we were unable to find/obtain a Conan
  executable.
Call Stack (most recent call first):
  build/_pmm/1.3.0/main.cmake:32 (_pmm_conan)
  build/_pmm/1.3.0/main.cmake:47 (_pmm_project_fn)
  CMakeLists.txt:34 (pmm)
```

I would recommend adding python3 to the list of dependencies so that the script will automatically install it to resolve the issue.

--- a/setup
+++ b/setup
@@ -235,6 +235,7 @@ case "${unameOut}" in
                 check_homebrew cmake MISSING
                 check_homebrew flex MISSING
                 check_homebrew bison MISSING
+                check_homebrew python3 MISSING
                 MISSING_LIST=$(printf " %s" "${MISSING[@]:1}")
                 MISSING_LIST=${MISSING_LIST:1}
                 if [ ${#MISSING[@]} -gt 0 ]; then 

